ovs_performance.py

```diff
@@ -3699,6 +3699,8 @@ def main():
                         action="store_true")
     parser.add_argument("--run-vxlan-test",
```

Comment:
How about renaming this test altogether, and making two independent tests, so they can be run in sequence with a single run, i.e., --run-vxlan-pv-test and --run-vxlan-pvp-test?